src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/confirmpoattendance/ConfirmPoAttendanceMidEventHandler.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.confirmpoattendance;
+
+import static java.util.Objects.requireNonNull;
+import static net.logstash.logback.util.StringUtils.isEmpty;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.CaseDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Contact;
+import uk.gov.hmcts.reform.sscs.ccd.domain.EventType;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+import uk.gov.hmcts.reform.sscs.ccd.presubmit.PreSubmitCallbackHandler;
+
+@Component
+@Slf4j
+@AllArgsConstructor
+public class ConfirmPoAttendanceMidEventHandler implements PreSubmitCallbackHandler<SscsCaseData> {
+
+    @Override
+    public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callback) {
+        requireNonNull(callback, "callback must not be null");
+        requireNonNull(callbackType, "callbacktype must not be null");
+
+        return callbackType.equals(CallbackType.MID_EVENT)
+                && callback.getEvent() == EventType.CONFIRM_PO_ATTENDANCE;
+    }
+
+    @Override
+    public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType, Callback<SscsCaseData> callback,
+                                                          String userAuthorisation) {
+        if (!canHandle(callbackType, callback)) {
+            throw new IllegalStateException("Cannot handle callback");
+        }
+
+        final CaseDetails<SscsCaseData> caseDetails = callback.getCaseDetails();
+        SscsCaseData sscsCaseData = caseDetails.getCaseData();
+
+        PreSubmitCallbackResponse<SscsCaseData> response = new PreSubmitCallbackResponse<>(sscsCaseData);
+
+        Contact poContact = sscsCaseData.getPresentingOfficersDetails().getContact();
+        if (isEmpty(poContact.getMobile()) && isEmpty(poContact.getPhone())) {
```

Comment:
Any chance this poContact object can be null here? Is it worth adding null check?

Reply:
in common I have overridden the getter method to send back an empty object if null (same with the name and PoDetails objects)